subsys/net/lib/lwm2m/Kconfig

```
@@ -385,6 +385,13 @@ config LWM2M_RW_JSON_SUPPORT
 	help
 	  Include support for writing JSON data
 
+
+config LWM2M_RW_SENML_JSON_SUPPORT
+	bool "support for SENML JSON data format"
```

Minor nit, as per the guidelines, consider to just write: `SENML JSON data format`:
https://docs.zephyrproject.org/latest/guides/build/kconfig/tips.html#prompt-strings

> For a Kconfig symbol that enables a driver/subsystem FOO, consider having just “Foo” as the prompt, instead of “Enable Foo support” or the like.

```suggestion
	bool "SENML JSON data format"
```

---

OK I will fix that.